### PR TITLE
[[Dictionary]] Fixes to sort-container.lcdoc

### PR DESCRIPTION
**Pull request by asayd**
Fixed formatting in Parameters element
Fixed formatting of code block in Description element
Added references and links
Declared variables in examples

### CODE REVIEW

File: docs/notes/bugfix-20688 [hunk 1]
```diff
@@ -0,0 +1 @@
+# [Dictionary] "sortType" descriptlon, line 1 mistake
```
Comment (montegoulding) [inline docs/notes/bugfix-20688]: @asayd I think this should be something like

```
# Fix the formatting of the `sort` command `sortType` description
```

**Comment by montegoulding** [comment]
@livecode-vulcan review ok b1c4a9a

Thanks @asayd

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok b1c4a9a93c891d846b78bcbae9bab9fe534192eb

**Comment by livecode-vulcan** [comment]
😎 test success b1c4a9a93c891d846b78bcbae9bab9fe534192eb

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/999)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1015)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/989)
* try-community-universal-ios-iphoneos11.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.1/builds/62)
* try-community-universal-ios-iphonesimulator11.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.1/builds/62)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/994)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1016)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/999)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1015)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/998)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1008)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1031)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/995)

**Pull request merged**